src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        new User();
+        User user1;
+        user1 = UserService.findByEmail(email);
```

Comment:
```suggestion
        User user = userService.findByEmail(email);
```

Reply:
All fixes have been done. But there have left one mistake. Would you help me to understand the problem?
IntellijIdea proposes 'static'...

![image](https://user-images.githubusercontent.com/116544768/204553172-fa8761cf-868c-4136-9600-2f78ac3dc480.png)
![image](https://user-images.githubusercontent.com/116544768/204553937-e563b34e-23f5-403b-b73d-06310813406d.png)